src/package.json

```
@@ -55,6 +55,26 @@
     "bootstrap": "^3.3.7",
     "d3": "^5.9.2"
   },
+  "exports": {
+    "./styles/ux-aspects.css": {
+      "style": "./styles/ux-aspects.css"
+    },
+    "./styles/ux-aspects.min.css": {
+      "style": "./styles/ux-aspects.min.css"
+    },
```

I believe we can combine these:

```suggestion
    "./styles/*": {
      "style": "./styles/*.css"
    },
```